src/services/db/GitFileSystemService.ts

```diff
@@ -42,6 +37,19 @@ export default class GitFileSystemService {
     this.git = git
   }
 
+  isDefaultLogFields(logFields: unknown): logFields is DefaultLogFields {
+    const c = logFields as DefaultLogFields
+    return (
+      !!logFields &&
+      typeof logFields === "object" &&
```

Comment:
nit & non-blocking: we should use `c` in line 43 & 44 instead of `logFields` since it was already casted to `c` in line 41

Reply:
Will do separately in a hotfix PR